apps/mobile/src/components/RoundCheckBox/RoundCheckbox.js

```diff
@@ -1,17 +1,13 @@
 import React, { useEffect, useRef } from 'react'
 import { Animated, View, TouchableWithoutFeedback } from 'react-native'
 import Icon from 'react-native-vector-icons/Ionicons'
-import { caribbeanGreen, rhino50 } from 'style/colors'
 
 const RoundCheckbox = ({
   onValueChange,
   icon = 'checkmark-outline',
   size = 24,
-  checkedColor = caribbeanGreen,
-  uncheckedColor = rhino50,
-  iconColor = 'white',
-  checked = false,
-  style = {}
```

Comment:
I'd prefer we keep the `style` overrides. I've spend a few hours playing with different scenarios, and I don't see use using NativeWind for everything always and only. IMHO we should be, and will need to, support both `className` and `style`. I care about this in commonly used components (such as this), not so much in full/screen components where it is all domain specific.